### PR TITLE
Enforce that `Task.Backgroundable#run` is always `@UIThreadUnsafe` + get rid of unneeded background tasks

### PR DESCRIPTION
**Pull request by PawelLipski**
image

### CODE REVIEW

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/ShowSelectedInGitLogAction.java [hunk 1]
```diff
@@ -89,25 +83,12 @@ public void actionPerformed(AnActionEvent anActionEvent) {
...
-          LOG.error(msg);
-        }
-      }
-    }.queue();
+    val logUi = VcsProjectLog.getInstance(project).getMainLogUi();
+    if (logUi == null) {
+      LOG.error("Main VCS Log UI is null");
+      return;
+    }
+    logUi.getVcsLog().jumpToReference(hash.asString());
```
Comment (PawelLipski) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/ShowSelectedInGitLogAction.java:91]: This returns a Java `Future`. Still, we're free to ignore (fire-and-forget) that Future, just as we effectively ignore most `Task.Backgroundable`s after `queue()`.

Comment (mkondratek) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/ShowSelectedInGitLogAction.java:91]: no exception handling needed here? 🤔

**Review by mkondratek — APPROVED** [review]

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/ShowSelectedInGitLogAction.java (hunk 1 above)
Comment (PawelLipski) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/ShowSelectedInGitLogAction.java:91]: I don't think it's needed TBH. It's pretty hard for exception to occur here (I've checked, and passing `lolxd` results in just a tooltip; passing `null` crashes the flow before the `Future` starts). Even if it happens... it's gonna most likely be reported to the user (who hopefully reports it to us, in turn)

Comment (mkondratek) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/ShowSelectedInGitLogAction.java:91]: ok,thx

**Pull request merged**